content/desktop/synchronized-file-sharing.md

```diff
@@ -51,6 +51,12 @@ When the status indicator displays **Watching for filesystem changes**, your fil
 >
 > When you create a new service, setting the [bind mount option consistency](../reference/cli/docker/service/create.md#options-for-bind-mounts) to `:consistent` bypasses Synchronized file shares. 
 
+> **Tip**
+>
+> Compose can now automatically create file shares for bind mounts. 
+> Ensure you're signed into Docker with a paid subscription and have **Access experimental features** enabled in Docker Desktop settings.
+{ .tip }
```

Comment:
Maybe it's because I'm nightly build, but I don't see anything in the UI copy about this feature 
<img width="452" alt="image" src="https://github.com/docker/docs/assets/35727626/3bc3b965-1d76-423a-a543-0f8d2d812986">